job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceEndToEndIT.java

```diff
@@ -976,13 +976,13 @@ public void testJobServiceCaller_Success() throws ParseException, IOException, T
             long statusCode = (long) waitContainerResponse.get("StatusCode");
 
             //  Expecting StatusCode=0 for success.
-            Assert.assertNotNull(statusCode);
-            Assert.assertEquals(0L, statusCode);
+            Assert.assertNotNull(statusCode, "Job "+jobId+" status code is null");
+            Assert.assertEquals(statusCode, 0L, "Status code for Job "+jobId+" is not 0.");
```

Comment:
Why have statusCode and 0L been swapped around?

Reply:
This was a TestNG assertion
Assert.assertEquals(actual, expected, message) and 0L was an expected value and statusCode was actual. So I swapped to correct it.